tests/Unit/View/Components/SelectTest.php

```diff
@@ -8,12 +8,12 @@
 class SelectTest extends TestCase
 {
     /** @test */
-    public function it_should_throw_an_error_if_async_data_and_options_is_set_together()
```

Comment:
Instead of commenting out the test, use the `markTestSkipped` function to flag it as skipped.

Reply:
I fix this test in PR https://github.com/wireui/wireui/pull/638/files.

Reply:
And I had already done the Laravel Upgrade to version 10 in this PR too.